src/extension.ts

```diff
@@ -19,6 +20,26 @@ export function activate(context: vscode.ExtensionContext) {
       "Configuration": {}
     });
   };
+  const importsCommand = async (editor, edit, {completion}) => {
+    setTimeout(async () => {
+      try {
+        let selection = editor.selection;
+        let completionSelection = new vscode.Selection(selection.active.translate(0, -completion.length), selection.active);
+        let codeActionCommands = await vscode.commands.executeCommand<vscode.CodeAction[]>('vscode.executeCodeActionProvider', editor.document.uri, completionSelection, vscode.CodeActionKind.QuickFix);
+        let importCommands = codeActionCommands.filter(c => c.title.toLocaleLowerCase().includes("import"));
```

Comment:
Haven't run this code, but is it maybe possible that the `kind` of the `CodeAction` is more specific than `QuickFix`, allowing us to filter it by the `kind` rather than by title? If its possible I think that would be preferable. I Can't think of another code action which will include 'import' but if we can be more explicit with `kind` filtering, that sounds better to me.

Reply:
all the`QuickFix`s (add missing imports, add single import, add existing , etc. ) have the same kind of `vscode.CodeActionKind.QuickFix` and i already filtered them to match only the `vscode.CodeActionKind.QuickFix` by passing  the `kind` to `executeCodeActionProvider` command (the last argument)  

- `Import [module]`
- `add [module] to existing import`
(hence the lowercase check)
maybe we should explicitly filter for those two (with regex or something), WDYT?

Reply:
I'm in for documenting this, and I think that using regexes is a really good idea to do so in code, I vote for it. As far as I remember its also pretty performant in JS compared to string ops (indexOf etc.)

Reply:
done 👍